roblox/assets.py

```diff
@@ -95,8 +95,8 @@ class AssetType:
     Represents a Roblox asset type.
 
     Attributes:
-        id: Id of the Asset
-        name: Name of the Asset
```

Comment:
I think we should always capitalize "ID".

Reply:
Done